scripts/run-all-checks.sh

```
@@ -0,0 +1,334 @@
+
+    print_section "$check_name"
+
+    cd "$PROJECT_ROOT"
+
+    if [ ! -f "$VENV/bin/activate" ]; then
+        print_error "Virtual environment not found at venv/"
+        [ -n "$status_file" ] && echo "Code - Virtual environment not found" >> "$status_file"
+        return 1
+    fi
+
+    source "$VENV/bin/activate"
```

Shellcheck (0.11.0)

[info] Not following: ./bin/activate was not specified as input (see shellcheck -x).

(SC1091)